src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -39,12 +45,18 @@ public Optional<Movie> get(Long id) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             return Optional.ofNullable(session.get(Movie.class, id));
         } catch (Exception e) {
-            throw new DataProcessingException("Can't get a movie by id: " + id, e);
+            throw new DataProcessingException(CAN_T_GET_A_MOVIE_BY_ID_MSG + id, e);
         }
     }
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> fromMovie =
+                    session.createQuery("FROM Movie ", Movie.class);
+            return fromMovie.getResultList();
+        } catch (RuntimeException e) {
```

Comment:
In the `getAll` method, the catch block handles `RuntimeException`, while the other methods handle `Exception`. For consistency, consider handling `Exception` in the `getAll` method as well, unless there is a specific reason to handle only `RuntimeException`.